data/labels.go

```diff
@@ -80,6 +81,33 @@ func (l Labels) String() string {
 	return sb.String()
 }
 
+type Fingerprint uint64
```

Comment:
why the alias?

Reply:
Few reasons:
- standard type is too general and may be confusing to read in the code when passed around in user code. 
- simpler to track where the type\value originated. 
- add the possibility to extend the type with methods. For example, Stringer interface.
- improves maintainability. If we decide to switch from uint64 to byte array or string, less code will need to be rewritten.

Reply:
Also, I added String() to represent the hash as a sequence of hex numbers